README.md

```diff
@@ -10,10 +10,12 @@ In your git repository, create a file `.github/workflows/chromatic.yml` with the
 
 ```yml
 name: 'Chromatic'
-on: push
+on:
+  push:
+    branches: [ master ]
 
 jobs:
-  test:
+  deploy-storybook:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
```

Comment:
@GMaiolo we have a little "breathing room" (pardon the bad pun) and we could probably go with:
```yml
jobs:
    chromatic-deployment:
         runs-on: ubuntu-latest
         steps:
            - uses:actions/checkout@v1
```
Reasoning behind this is that publishing Storybook one part of the task. We're also doing Chromatic's automation process and setting visual regression testing.

Let me know if you're ok with

Reply:
No worries @jonniebigodes
I just updated it

Reply:
@GMaiolo fantastic. Once again thank your making the changes so promptly. From my end i'm good. If @ndelangen signs up on it we'll merge it.

Stay safe